src/wasm-interpreter.h

```diff
@@ -3431,7 +3431,7 @@ class ModuleRunnerBase : public ExpressionRunner<SubType> {
     if (timeout.getSingleValue().getInteger() != 0) {
       hostLimit("threads support");
     }
-    return Literal(int32_t(0)); // equal
+    return Literal(int32_t(2)); // Timed out
```

Comment:
I don't follow - if there is no timeout, how can we timeout?

Reply:
The timeout is zero nanoseconds.

Reply:
Oh, I see, reading the spec more carefully indeed 0 is just a brief timeout in effect...